assets/src/application/ui/layout/entityList/filterBar/filterState/useFilteredEntities.ts

```diff
@@ -39,7 +39,7 @@ const useFilteredEntities = <D extends string, L extends string, E extends Entit
 	// paginate it
 	cacheIds = entityListCacheIdString(sortedEntities);
 	const paginatedEntities = useMemo<Array<E>>(() => {
-		return sortedEntities.slice(perPage * (pageNumber - 1), perPage * pageNumber);
+		return sortingEnabled ? sortedEntities : sortedEntities.slice(perPage * (pageNumber - 1), perPage * pageNumber);
```

Comment:
what does `sortedEntities.slice(perPage * (pageNumber - 1), perPage * pageNumber` mean?
can this be extracted in variable?
there is too much logic in a single line

Reply:
That gets the entities for the current page based on `perPage` and `pageNumber`.

Reply:
Improved readability in that hook